python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -948,7 +997,64 @@ def reset_index(
             self.index = cudf_series.index
             return None
 
-    def contains_properly(self, other, align=False, allpairs=False):
+    def contains(self, other, align=False, allpairs=False, mode="full"):
+        """Returns a `Series` of `dtype('bool')` with value `True` for each
+        aligned geometry that contains _other_.
+
+        An object `a` is said to contain `b` if `b`'s `boundary` and
+        `interiors` are within those of `a` and no point of `b` lies in the
+        exterior of `a`.
+
+        If `allpairs=False`, the result will be a `Series` of `dtype('bool')`.
+        If `allpairs=True`, the result will be a `DataFrame` containing two
+        columns, `point_indices` and `polygon_indices`, each of which is a
+        `Series` of `dtype('int32')`. The `point_indices` `Series` contains
+        the indices of the points in the right GeoSeries, and the
+        `polygon_indices` `Series` contains the indices of the polygons in the
+        left GeoSeries.
+
+        Notes
+        -----
+        `allpairs=True` excludes geometries that contain points in the
+        boundary of A.
+
+        Parameters
+        ----------
+        other : GeoSeries
+        align : bool, default False
+            If True, the two GeoSeries are aligned before performing the
+            operation. If False, the operation is performed on the
+            unaligned GeoSeries.
+        allpairs : bool, default False
+            If True, the result will be a `DataFrame` containing two
+            columns, `point_indices` and `polygon_indices`, each of which is a
+            `Series` of `dtype('int32')`. The `point_indices` `Series` contains
+            the indices of the points in the right GeoSeries, and the
+            `polygon_indices` `Series` contains the indices of the polygons in
+            the left GeoSeries. Excludes boundary points.
+        mode : str, default "full"
```

Comment:
List all available `str` on the brief description, so that user don't have read the below to remember config options.